libs/ngworker/lumberjack/src/lib/formatting/lumberjack-format-log.spec.ts

```diff
@@ -0,0 +1,134 @@
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import { resolveDependency } from '@internal/test-util';
+
+import { LumberjackLogFactory } from '../logging/lumberjack-log-factory';
+import { LumberjackLogBuilder } from '../logging/lumberjack-log.builder';
+import { LumberjackLevel } from '../logs/lumberjack-level';
+import { LumberjackLogLevel } from '../logs/lumberjack-log-level';
+import { LumberjackLog } from '../logs/lumberjack.log';
+import { LumberjackTimeService } from '../time/lumberjack-time.service';
+
+import { lumberjackFormatLog } from './lumberjack-format-log';
+
+function parseFormattedLog(formattedLog: string) {
+  // eslint-disable-next-line no-useless-escape
+  const formattedLogPattern = /^([a-z]+) ([0-9\.:\-TZ]+) (\[(.+)\] )?(.*)$/;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [_, level, timestamp, taggedScopeWithEndingSpace = '', scope = '', message] =
```

Comment:
suggestion: We might remove this lint rule deactivation by leaving out the underscore assignment.
```suggestion
  const [, level, timestamp, taggedScopeWithEndingSpace = '', scope = '', message] =
```